### PR TITLE
meta: Improve code owners for SDKs and CLI

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -5,14 +5,18 @@
...
+/src/platforms/java/ @bruno-garcia @marandaneto
 
 /src/platforms/native/ @getsentry/owners-native
 
 /src/platforms/rust/ @Swatinem
 
 /src/docs/product/discover-queries/ @getsentry/visibility
 /src/docs/product/performance/ @getsentry/visibility
+
+/src/docs/product/cli/ @getsentry/team-webplatform
```
Comment (jan-auer) [inline .github/CODEOWNERS:21]: This is probably a bit much, how about we restrict this to the releases commands?

If you don't mind, I would add the native team in the same PR?

Comment (marandaneto) [reply .github/CODEOWNERS:21]: @jan-auer sure, go for it.
pay attention that the sorting here matters, the last match is the one that will be assigned as reviewer.

Comment (jan-auer) [reply .github/CODEOWNERS:21]: Ok so the problem is that I cannot select @getsentry/owners-native as a reviewer team in sentry-docs. Probably a permission issue. 

@PeloWriter to you have admin on this repo to look into this?

**Review by bruno-garcia — APPROVED** [review]
From my end: 👍

File: .github/CODEOWNERS (hunk 1 above)
Comment (PeloWriter) [reply .github/CODEOWNERS:21]: @jan-auer - owners-native needs to be granted permissions to the docs repo, which should be handled by an IT request (for compliance/security reasons).

Comment (jan-auer) [reply .github/CODEOWNERS:21]: We're probably going to address this by moving the @getsentry/codeowners team under @getsentry/engineering, which should take care of this.

Comment (PeloWriter) [reply .github/CODEOWNERS:21]: 👍

**Review by jan-auer — APPROVED** [review]
Requested review from @kamilogorek since he is now added as owner for CLI.

**Review by kamilogorek — APPROVED** [review]

**Comment by kamilogorek** [comment]
image

**Pull request merged**